yarn-project/end-to-end/src/e2e_blacklist_token_contract/minting.test.ts

```diff
@@ -83,21 +83,18 @@ describe('e2e_blacklist_token_contract mint', () => {
     });
 
     describe('Mint flow', () => {
-      it('mint_private as minter', async () => {
+      it('mint_private as minter and redeem as recipient', async () => {
```

Comment:
Had to merge these 2 tests as now it's not supported by the token simulator to have the shield flow in 2 steps.